src/main/java/org/trips/service_framework/services/CmsService.java

```diff
@@ -136,15 +137,14 @@ public Sku getSkuByAttributes(SkuAttributes attributes) {
         }
 
         if (response.getData().searchSkus.size() > 1) {
-            log.error("Multiple SKUs found for the given sku attributes");
-            throw new CmsException("Multiple SKUs found for the given sku attributes");
+            log.error("Multiple SKUs found for the given sku attributes, returning the first SKU");
         }
 
         return response.getData().searchSkus.get(0);
     }
 
     public Sku createSku(SkuAttributes skuAttributes) {
-        ValidationUtils.validate(skuAttributes);
+        ValidationUtils.validate(CmsUtils.validateQuantityAttributes(skuAttributes));
```

Comment:
this is not a validation but an enrichment of the pojo. IMO, it should be renamed to CmsUtils.enrichQuantityAttributes. @dagaranupam @hardikphalet